src/Wallet.js

```diff
@@ -41,6 +41,40 @@ exports._submitTxNami = txHex => nami => () => {
       });
 };
 
+// _enableGero :: Effect (Promise NamiConnection)
+exports._enableGero = () => window.cardano.gero.enable();
```

Comment:
Is this the correct API object? Trying to run `Examples.Gero` gives me:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'enable')
    at exports._enableGero (output.js?ee2e:9826:1)
    at runSync (output.js?ee2e:6816:1)
    at run (output.js?ee2e:7054:1)
    at eval (output.js?ee2e:7365:1)
    at drain (output.js?ee2e:6846:1)
    at Object.enqueue (output.js?ee2e:6867:1)
    at Object.run (output.js?ee2e:7364:1)
    at __do (output.js?ee2e:7970:1)
    at eval (output.js?ee2e:6526:1)
    at Module.eval (output.js?ee2e:6526:1)
exports._enableGero @ output.js?ee2e:9826
runSync @ output.js?ee2e:6816
run @ output.js?ee2e:7054
eval @ output.js?ee2e:7365
drain @ output.js?ee2e:6846
enqueue @ output.js?ee2e:6867
run @ output.js?ee2e:7364
__do @ output.js?ee2e:7970
eval @ output.js?ee2e:6526
eval @ output.js?ee2e:6526
eval @ index.js?2414:8
setTimeout (async)
run @ output.js?ee2e:7254
eval @ output.js?ee2e:7365
drain @ output.js?ee2e:6846
enqueue @ output.js?ee2e:6867
run @ output.js?ee2e:7364
__do @ output.js?ee2e:7970
eval @ output.js?ee2e:6526
eval @ output.js?ee2e:6526
eval @ index.js?2414:8
Promise.then (async)
eval @ index.js?2414:8
./examples/index.js @ bundle.js:17
__webpack_require__ @ bundle.js:656
(anonymous) @ bundle.js:2106
```

Reply:
Actually, it should be `window.cardano.gerowallet.enable()`

Will fix in the next set of commits

Reply:
Fixed, reopen if any errors persist